stylegan_code_finder/train_with_lightning.py

```diff
@@ -33,18 +28,8 @@ def sanity_check_config(config: dict):
                                                              f'be equal to the num_classes in the config'
 
 
-def setup_distributed(mpi_backend: str, rank: int, world_size: int):
-    os.environ['MASTER_ADDR'] = 'localhost'
-    os.environ['MASTER_PORT'] = '12355'
-    dist.init_process_group(mpi_backend, rank=rank, world_size=world_size)
-
-
-def cleanup():
-    dist.destroy_process_group()
-
-
-def main(rank: int, args: argparse.Namespace, world_size: int):
-    if world_size == 1 and os.environ.get('REMOTE_PYCHARM_DEBUG_SESSION', False):
+def main(args: argparse.Namespace):
+    if os.environ.get('REMOTE_PYCHARM_DEBUG_SESSION', False):
```

Comment:
I would probably move this to the top again or find another mechanism for debugging distributed lightning. `world_size == 1` was to make sure that only one process tries to connect to the debugger otherwise there were errors